### DISCUSSION TITLE
What was the reason for removing the xnack support for all rdna2+ cards?

### DISCUSSION THREAD

**Discussion by bog-dan-ro** [ROCm/ROCm#2867] [General]
Hi,

According to https://github.com/ROCm/ROCR-Runtime/blob/master/src/core/runtime/isa.cpp#L339 all RDNA2+ (including the APUs e.g. gfx1036) have no xnack suport. Is it not supported yet? Or was it removed for good?
If it was removed for good, then was it replaced with something better?
I'm asking because I have the feeling that am I missing something here, as xnack was a great feature (Unified Shared Memory) and I don't understand why it was removed for all RDNA2+.

**Comment by littlewu2508** [comment]
Maybe https://elixir.bootlin.com/linux/v6.8.3/source/drivers/gpu/drm/amd/amdkfd/kfd_process.c#L1431 helps
